include/crow/app.h

```diff
@@ -311,6 +318,7 @@ namespace crow
     private:
         uint16_t port_ = 80;
         uint16_t concurrency_ = 1;
+        std::string server_name_ = "Crow/0.2";
         std::string bindaddr_ = "0.0.0.0";
```

Comment:
any specific reason for not initializing them in ctor?

Reply:
@ayaankhan98 if you look at all the other methods and private variables, they are written the same.

Reply:
> @ayaankhan98 if you look at all the other methods and private variables, they are written the same.

yes, my intention was not to modify this in this PR, but just asking is there any specific reason for this or it requires modification in near future.

Reply:
it seems to be a better option than initializing in the ctor, since we know we need default values for these variables. That way, no matter how many ctors we have we don't need to copy paste these values.

Reply:
got it, Thanks!